code/iaas/model/src/main/resources/META-INF/cattle/core-model/spring-core-model-context.xml

```diff
@@ -33,6 +33,7 @@
             <set>
                 <value>virtualMachine,parent=instance</value>
                 <value>container,parent=instance</value>
+                <value>containerLogsInput</value>
```

Comment:
You should be registering the class ContainerLogs, not the string type.  Add it at https://github.com/rancherio/cattle/pull/210/files#diff-c619be8811e6d62c27a8064d172fb143R19